src/components/ModelDialog/ModalDialog.test.tsx

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
The title of this folder is Model instead of Modal

Reply:
Made a new folder with the correct name and moved all files to there.